serialize.go

```diff
@@ -0,0 +1,17 @@
+package hessian
+
+type Serializer interface {
+	serializeObject(*Encoder, POJO) error
+	deserializeObject(*Decoder) (interface{}, error)
+}
+
+var CodecMap = make(map[string]Serializer, 16)
+
+func SetSerializer(key string, codec Serializer) {
```

Comment:
export the methods and rename to `Serialize` and `Deserialize`, for the serializer may be defined in other package by user.

Reply:
this is store Serializer, it will set or get a Serializer.